.gitignore

```diff
@@ -50,3 +50,6 @@ bin/release/aws-eb/metabase-aws-eb.zip
 /locales/metabase-*.pot
 /stats.json
 coverage-summary.json
+.DS_Store
+bin/node_modules/
+yarn-error.log
```

Comment:
I might consider changing this to `*.log.` I can't imagine there's any `.log` files we want to keep in git history.

Reply:
Makes sense to me. Done.